src/api/types/generated-schema.ts

```diff
@@ -7260,7 +7261,7 @@ export interface components {
             /** Format: int32 */
             plantingDensityStdDev?: number;
             /** Format: int64 */
-            plantingSubzoneId: number;
+            plantingSubzoneId?: number;
```

Comment:
Nullability of the subzone and zone IDs hasn't actually changed in the API yet (even with https://github.com/terraware/terraware-server/pull/2968) but it will change in an upcoming server PR. I made the change here so I could verify that the zone/subzone IDs becoming nullable wouldn't cause type-checking errors on the client side.